theme-dark.scss

```diff
@@ -5,4 +5,5 @@ $navbar-fg: #f9bf07;
 
 $body-bg: #2c2825;
 $body-fg: #f5f4f3;
+$body-color: #f5f4f3;
```

Comment:
This is much better. Can we also have the code-block backgrounds (`$code-block-bg`) darker to add contrast as well? A bit darker than `$body-bg` I reckon.